### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by hubicios**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,30 +87,46 @@ python src\dataset_preparation\main.py
...
 Replace "HYPERKVASIR_PATH" and “ERS_PATH” with the path to your input images and "OUTPUT_PATH" with the path where you want to save the output sets. Use the optional arguments to specify the type of training, size of train, test and validation sets and various other options.
 
-More information presented in help option after running below command from project root
+e.g command to prepare data for [FCBFormer](https://github.com/ESandML/FCBFormer)
+```
+--ers-path "/raid/gwo/public/gastro/ers" --output-path "./fcb-ers" -f --binary --train-size 1.0 --test-size 0.0 --ers-class-mapper-path "mappers/2-class-polyp.yaml" --ers-use-seq --mask-mode "RGB" --path-ignore-dataset-name --path-ignore-dataset-type
+```
+e.g. command to prepare data for [ESFPNet](https://github.com/dumyCq/ESFPNet)
+```
+--ers-path "./ers" -f --binary  --train-size=0.8 --test-size=0.2 --ers-class-mapper-path "./mappers/2-class-polyp.yaml" --ers-use-seq  --training-type segmentation
```
Comment (wrobcio789) [inline README.md]: Lacking python main.py

File: README.md [hunk 2]
```diff
@@ -87,30 +87,46 @@ python src\dataset_preparation\main.py
 
 Replace "HYPERKVASIR_PATH" and “ERS_PATH” with the path to your input images and "OUTPUT_PATH" with the path where you want to save the output sets. Use the optional arguments to specify the type of training, size of train, test and validation sets and various other options.
 
-More information presented in help option after running below command from project root
+e.g command to prepare data for [FCBFormer](https://github.com/ESandML/FCBFormer)
+```
+--ers-path "/raid/gwo/public/gastro/ers" --output-path "./fcb-ers" -f --binary --train-size 1.0 --test-size 0.0 --ers-class-mapper-path "mappers/2-class-polyp.yaml" --ers-use-seq --mask-mode "RGB" --path-ignore-dataset-name --path-ignore-dataset-type
```
Comment (wrobcio789) [inline README.md]: Lacking python main.py

File: README.md [hunk 3]
```diff
@@ -87,30 +87,46 @@ python src\dataset_preparation\main.py
...
+```
+e.g. command to prepare data for [ESFPNet](https://github.com/dumyCq/ESFPNet)
+```
+--ers-path "./ers" -f --binary  --train-size=0.8 --test-size=0.2 --ers-class-mapper-path "./mappers/2-class-polyp.yaml" --ers-use-seq  --training-type segmentation
+```
 
+More information presented in help option after running below command from project root  
 ```
 python src\dataset_preparation\main.py --help
```
Comment (wrobcio789) [inline README.md]: Old command

File: README.md [hunk 4]
```diff
@@ -87,30 +87,46 @@ python src\dataset_preparation\main.py
...
+without flag → test/ers/images/polyp/1.png
+- `--path-ignore-dataset-name`  
 Flag specyfing whether the output path should ignore dataset name (examples: hyperkvasir/ers)  
-- `--path-ignore-img-type PATH_IGNORE_IMG_TYPE`  
-Flag specyfing whether the output path should contain type of image (images/masks)  
-- `--path-ignore-class-name PATH_IGNORE_CLASS_NAME`  
-Flag specyfing whether the output path should contain class name (polyp/ulcer)  
+e.g. with flag → test/images/polyp/1.png  
+without flag → test/ers/images/polyp/1.png
+- `--path-ignore-img-type`
```
Comment (wrobcio789) [inline README.md]: Please remove support for this flag. It's stupid

**Review by wrobcio789 — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (hubicios) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (hubicios) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (hubicios) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (hubicios) [reply README.md]: done

**Pull request merged**